src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="inputs">
+      <form
+        class="inputs__form"
+        data-qa="big"
+      >
+        <input
+          class="inputs__bigInput"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <form
+        class="inputs__form"
+        data-qa="small"
+      >
```

Comment:
The `action` attribute is missing for this form. If the form is meant to perform a specific action, please ensure to set the correct `action` attribute.